dotcom-rendering/src/experiments/tests/mobile-discussion-ads.ts

```diff
@@ -0,0 +1,28 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const mobileDiscussionAds: ABTest = {
+	id: 'MobileDiscussionAds',
+	author: '@commercial-dev',
+	start: '2024-02-05',
+	expiry: '2024-03-05',
+	audience: 0 / 100,
+	audienceOffset: 0 / 100,
```

Comment:
👀

Reply:
Mentioned in the PR description - we have yet to confirm sample size with D&I